src/components/ens-profile/ProfileSheetHeader.tsx

```diff
@@ -50,6 +54,25 @@ export default function ProfileSheetHeader({
     [profileAddress]
   );
 
+  const handleDescription = useCallback(() => {
+    const recordsDescription = profile?.records?.description;
+    if (!recordsDescription) return null;
+    const ensNames = recordsDescription?.match(ENS_REGEX);
+    const text = recordsDescription?.split(ENS_REGEX);
+    return (
+      <Inline alignVertical="center">
+        {text?.map((t, i) => (
+          <>
+            <Text containsEmoji key={i} weight="medium">
+              {t}
+            </Text>
+            {ensNames?.[i] && <Hyperlink value={ensNames?.[i]} />}
```

Comment:
Do we also want to hyperlink normal URLs too? Think that would happen more often.